easybuild/easyconfigs/p/POT/POT-0.9.4-foss-2023a.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonBundle'
+
+name = 'POT'
+version = '0.9.4'
```

Comment:
@PetrKralCZ As discussed, POT 0.9.4 requires `numpy>=2.0`, which we don't have yet, so we should stick to POT 0.9.3 here...